frontend/src/store/GrantAccessDataState.ts

```diff
@@ -4,6 +4,7 @@ import type { FamUserRoleAssignmentCreate } from 'fam-api';
 export const domainOptions = { IDIR: 'I', BCEID: 'B' }; // TODO, load it from backend when backend has the endpoint.
 
 export const grantAccessFormData = ref<FamUserRoleAssignmentCreate | null>();
+export const roleNameData = ref<String>();
```

Comment:
I wonder why we need a separate roleNameData? The grantAccessFormData should have the role data?